desktop/lib.go

```diff
@@ -85,7 +85,7 @@ func start() {
 	settings := loadSettings(cdir)
 	webclientOpts := &webclient.Opts{
 		HttpClient: &http.Client{
-			Transport: proxied.ParallelPreferChained(),
+			Transport: proxied.ChainedThenFronted(),
```

Comment:
@atavism I had changed this due to an error, You can check Slack more about that, For now even using proxied.ChainedThenFronted gives error on the first request.